vaadin-card-flow-parent/vaadin-card-testbench/src/main/java/com/vaadin/flow/component/card/testbench/CardElement.java

```diff
@@ -23,4 +29,102 @@
  */
 @Element("vaadin-card")
 public class CardElement extends TestBenchElement {
+
+    /**
+     * Gets the content elements.
+     *
+     * @return the content elements
+     */
+    public List<TestBenchElement> getContents() {
+        try {
+            var contentWrapper = findElement(By.cssSelector("div:not([slot])"));
+            return contentWrapper.getPropertyElements("children");
+        } catch (NoSuchElementException e) {
+            return Collections.emptyList();
+        }
+    }
+
+    /**
+     * Gets the element in the title slot.
+     *
+     * @return the title element, or {@code null} if not found
+     */
+    public TestBenchElement getTitle() {
+        return getElementInSlot("title");
+    }
+
+    /**
+     * Gets the title property value.
+     *
+     * @return title property value
+     */
+    public String getStringTitle() {
+        return getPropertyString("title");
+    }
+
+    /**
+     * Gets the element in the subtitle slot.
+     *
+     * @return the subtitle element, or {@code null} if not found
+     */
+    public TestBenchElement getSubtitle() {
+        return getElementInSlot("subtitle");
+    }
+
+    /**
+     * Gets the element in the media slot.
+     *
+     * @return the media element, or {@code null} if not found
+     */
+    public TestBenchElement getMedia() {
+        return getElementInSlot("media");
+    }
+
+    /**
+     * Gets the element in the header slot.
+     *
+     * @return the title element, or {@code null} if not found
+     */
+    public TestBenchElement getHeader() {
+        return getElementInSlot("header");
+    }
+
+    /**
+     * Gets the element in the header prefix slot.
+     *
+     * @return the header prefix element, or {@code null} if not found
+     */
+    public TestBenchElement getHeaderPrefix() {
+        return getElementInSlot("header-prefix");
+    }
+
+    /**
+     * Gets the element in the header suffix slot.
+     *
+     * @return the header suffix element, or {@code null} if not found
+     */
+    public TestBenchElement getHeaderSuffix() {
+        return getElementInSlot("header-suffix");
+    }
+
+    /**
+     * Gets the elements in the footer slot.
+     *
+     * @return the footer elements
+     */
+    public List<TestBenchElement> getFooterContents() {
+        return wrapElements(
+                getWrappedElement()
+                        .findElements(By.cssSelector("[slot='footer']")),
+                getCommandExecutor());
```

Comment:
```suggestion
        return findElements(By.cssSelector("[slot='footer']")).stream()
                .map(el -> (TestBenchElement) el).toList();
```
`TestBenchElement.findElements` already wraps Selenium elements, it's just that the return type doesn't reflect that.